README.rst

```diff
@@ -142,7 +142,7 @@ zero-based integer will be used.
 Format of race history
 ----------------------
 
-wpm will save scores in a CSV file in `~/.wpm.csv`. This file can be loaded
+wpm will save scores in a CSV file in `~/.config/wpm/wpm.csv`. This file can be loaded
```

Comment:
I'd rather just use `wpm.csv` since this will (according to what I propose) be in different directories, dependent on what system and setup you have. Then add a short section where you describe how we decide where the file goes (simple: first time user and you have XDG_HOME, it goes there, otherwise your home directory).